conan/internal/model/conf.py

```diff
@@ -126,6 +126,7 @@
     "tools.apple:enable_arc": "(boolean) Enable/Disable ARC Apple Clang flags",
     "tools.apple:enable_visibility": "(boolean) Enable/Disable Visibility Apple Clang flags",
     "tools.env.virtualenv:powershell": "If specified, it generates PowerShell launchers (.ps1). Use this configuration setting the PowerShell executable you want to use (e.g., 'powershell.exe' or 'pwsh'). Setting it to True or False is deprecated as of Conan 2.11.0.",
+    "tools.env:dotenv": "Generate dotenv environment files",
```

Comment:
Should we make it an opt-in conf?
Or maybe generate it by default, always to make it simpler/more convenient?

Reply:
I like the idea that it would be nice to always generate this file with and opt-out, (but would we even want an specific opt-out?)

Reply:
We might wait for the opt-out until users give feedback.